admin/testadmin/testadmin.go

```diff
@@ -50,6 +53,23 @@ type Fixture struct {
 	ServerOpts *server.Options
 }
 
+// NewGithub creates a new Github client. In short testing mode this is a mock client which has no-op implementations of all methods.
+// Otherwise it creates a real implementation that makes real API calls to Github.
+func NewGithub(t *testing.T) admin.Github {
```

Comment:
1. Can you move this later in the file? It breaks the style guide on function ordering: https://github.com/uber-go/guide/blob/master/style.md#function-grouping-and-ordering
2. Since it's only used inside this package, should it be `newGithub`?